### ISSUE TITLE
Path to match relative (./*) imports

### ISSUE THREAD

**Issue by breandr** [jspm/jspm-cli#831]
I've been stuck on this all day. It started with my own imports in the format:

```
//app/components/method/method.js
import './method-service';
```

And when running `jspm bundle app/app app-bundle.js`, my system.config path wasn't recognising this with the rule:

```
"paths": {
    "*": "*.js"
}
```

So I changed all my relative imports to, for example:

```
import 'app/components/method/method-service';
```

This resolved my issue for all of my own code, however, I have a lot of third-party code that is using the "./" approach. Do you have any idea why this would be happening? I tried adding rules to paths `"./*": "*.js"` and `"./*": "./*.js"` but it still did not pick that up. Any suggestions would be greatly appreciated.

This is on a RHEL5 running NodeJS 0.10.

**Comment by guybedford** [comment]
Can you confirm if you are running the jspm beta here?

**Comment by breandr** [comment]
0.15.7. If you've never heard of this odd issue before, the problem is likely somewhere else. I have to call jspm indirectly. For example: `executor /full/path/to/jspm/jspm.js bundle ./app/app app-bundle.js`. If nothing comes to mind, please just close this.

I'll try to work around it by creating a Grunt task which will allow me to bypass the "executor". Do you know of a Grunt task that is similar to what I will need to do for grunt-jspm so that I can use it for reference?

**Comment by guybedford** [comment]
Right, jspm will seek the project package.json from the running directory, perhaps you need to ensure its detecting the package ok here? Did you try running `executor /full/path/to/jspm/jspm.js init` to confirm it's all ok first?

**Comment by guybedford** [comment]
(That is are you sure any config is being loaded?)

**Comment by breandr** [comment]
I haven't run init yet but I have confirmed it is loading config (it's
mapping github and npm correctly, just no ".js" being appended for things
imported/required as relative ./). I've put integrating this with Grunt in
my backlog for now. I've got it successfully bundling from a Mac so I just
have to do that manually until I get time to revisit this. Thanks for your
assistance and I'll report back if it turns out to be anything relevant to
jspm (also with any work I do on a Grunt plugin)
